src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,137 @@
+p,
+h2 {
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px 0px 0px 0px;
+  border: 1px solid #f3f3f3;
+  opacity: 0px;
+
+  padding: 32px 16px 16px;
+  margin: 0;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin: 0 auto 40px;
```

Comment:
The margin value `40px` seems quite specific. Make sure that this value matches the design requirements. If this is a spacing that is used multiple times across different elements, consider using a variable for consistency and easier maintainability.